### DISCUSSION TITLE
Release date for adaptive cards

### DISCUSSION THREAD

**Discussion by svalio** [microsoft/AdaptiveCards#5385] [Q&A]
I have a question about the Adaptive card schema version for MS Teams as mentioned in the documentation https://docs.microsoft.com/en-us/adaptive-cards/authoring-cards/input-validation there is isRequired option for inputs, but once I have tried this in the designer I see that version 1.3 currently is not supported for the MS Teams scope.
![image](https://user-images.githubusercontent.com/36510050/107648229-b69a9900-6c84-11eb-85ad-b35bc7da9ef3.png)

Is there a release date for the schema version 1.3 to support MS Teams?

**Comment by mkulgeorge** [comment]
Addition of this feature to MS teams will help the developers a lot as we don't have to write the input validation in the code and circle back to user, reduces lots of validation coding. So every eager to see this option to be available in MS Teams.
